Source/JavaScriptCore/wasm/WasmTable.h

```diff
@@ -52,8 +51,18 @@ class Table : public ThreadSafeRefCounted<Table> {
     WTF_MAKE_NONCOPYABLE(Table);
     WTF_MAKE_TZONE_ALLOCATED(Table);
 public:
+    using MallocOps = VMMalloc;
+
     static RefPtr<Table> tryCreate(uint32_t initial, std::optional<uint32_t> maximum, TableElementType, Type);
 
+    template<typename T>
+    struct StorageDeleter {
+        size_t count { 0 };
+        bool ownsStorage { false };
+
+        void operator()(T* ptr);
+    };
+
     JS_EXPORT_PRIVATE ~Table() = default;
```

Comment:
Can we move this line next to the `operator delete` overload. It would make it a bit clearer what's going on.